# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/mtransport/third_party/nICEr/src/net/transport_addr.h
# Commit: f18e61be455d
# Full Hash: f18e61be455d493f10a2cc651cd254859894e02e
# Author: Byron Campen [:bwc] <docfaraday@gmail.com>
# Date: 2019-02-26 03:52:35
# Regressor Bug: 1521879
# File Overlap Count: 1
# Description:
#   Bug 1521879 - Part 1: IPC-based MediaTransport implementation r=mjf
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D17273
# ==============================================================================

diff -r 8136a17fbec9 -r f18e61be455d media/mtransport/third_party/nICEr/src/net/transport_addr.h
--- a/media/mtransport/third_party/nICEr/src/net/transport_addr.h	Mon Feb 25 21:48:20 2019 +0000
+++ b/media/mtransport/third_party/nICEr/src/net/transport_addr.h	Mon Feb 25 21:50:42 2019 +0000
@@ -44,6 +44,8 @@
 #include <netinet/in.h>
 #endif
 
+#include "r_types.h"
+
 /* Length of a string  hex representation of a MD5 hash */
 #define MAXIFNAME 33
 